src/resources/clientPolicies.ts

```diff
@@ -16,12 +17,20 @@ export class ClientPolicies extends Resource<{realm?: string}> {
     });
   }
 
-  public listProfiles = this.makeRequest<{includeGlobalProfiles?: boolean}, ClientProfilesRepresentation>({
+  public listProfiles = this.makeRequest<
+    {includeGlobalProfiles?: boolean},
+    ClientProfilesRepresentation
+  >({
```

Comment:
Is there are reason why the formatting was changed here?

Reply:
TSLint autoformats to this on save